README.md

```diff
@@ -45,6 +43,10 @@ We distribute a `docker-compose.yml` configuration for running the Hyku stack an
 docker-compose up -d
 ```
 
+### With Vagrant
+
+The [samvera-vagrant project](https://github.com/samvera-labs/samvera-vagrant) provides another simple way to get started "kicking the tires" of Hyku (and [Hyrax](http://hyr.ax/), making it easy and quick to spin up Hyku. (Note that this is not for production or production-like installations.) It requires VirtualBox and Vagrant.
+
```

Comment:
Might need an extra closing `)` to complete the parens for `(and [Hyrax](http://hyr.ax/))`.